2-aragon-network-dao-agreement.tex

```diff
@@ -65,7 +65,7 @@ \subsection{Proposal Compliance}
 
 \subsection{Disputes}
 \begin{enumerate}
-	\item Disputes over any proposal relying on this Agreement must be resolved by \gls{AragonCourt}.
+	\item Disputes over any proposal relying on this Agreement must be resolved by \gls{TechAgnosticMechanisms}.
```

Comment:
In my opinion, words that specify a concrete tool such as 'Aragon Voice' or 'Aragon Court' can't simply be replaced by the phrase 'tech agnostic mechanism'. 
'Tech agnostic mechanism' specifies nothing (except that there is some form of mechanism) so that people can theoretically flip a coin instead of using Aragon Voice or Aragon Court.

More generally, I also don’t understand how a DAO living on a blockchain and using web3 technology can have a tech agnostic charter. Do we mean vendor-agnostic?

Instead of naming a specific tool/vendor such as Aragon Voice, we can generalize this by naming the characteristics and requirements (e.g., 'a percentage-based decision-making tool with properties X, Y, and Z'). 
The proposers / people motivating this charter change should precise what the characteristics and requirements are. 

Example: 'a trustless, sybil-resistant, percentage-based decision-making tool for signalling purposes with the possibility for on-chain execution'.

The same line of argument holds for Aragon Court, which cannot simply be replaced by 'tech agnostic mechanism'.

Aragon Court as well as Kleros Court and Celeste are decentralized arbitration services. If we don’t want to have arbitration processes in the charter, we have to specify concrete alternative processes that replace them.